### PR TITLE
Change certificates to support net.tcp scenarios

### PR DESCRIPTION
**Pull request by iamjasonp**
Renamed cert private keys/public cert files to be less confusing
Updated README.txt to README.md

This change is to make the certs play better with the net.tcp transport security tests. Previously, the certificates used ran into an issue with the net.tcp test - an exception was thrown stating that the key was not usable for key exchange.

### CODE REVIEW

**Comment by iamjasonp** [comment]
(Part of #310, but #310 isn't fixed yet.)

**Comment by hongdai** [comment]
Built and run the outloop tests on my local machine successfully. LGTM.

**Comment by iamjasonp** [comment]
Please hold off on merging, we need to reduce the expiry time on the new root cert.

**Comment by iamjasonp** [comment]
Ready for merging, please do a buddy build with [OuterLoop] tests prior to doing so though.

**Comment by roncain** [comment]
@iamjasonp You will need to update rc\System.Private.ServiceModel\tests\Common\Infrastructure\src\testproperties.props to use the new names.  These TestProperties are passed verbatim to the Bridge during the /config setup and are how the Bridge locates the certificates in the BridgeResourceFolder.

File: src/System.Private.ServiceModel/tools/test/SelfHostWcfService/Certificates/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+```
+    makecert -n "CN=<<CanonicalName>>" -pe -r -sky exchange -cy authority -sk PrivateKey -eku <<Enhanced key usages>> -ss <<Subject cert store>> -sr <<Subject cert store location>> -m <<months to expiry>> <<filename>> 
+```
+
+Actual:
+
+```
+    makecert -n "CN=DO_NOT_TRUST_WcfBridgeRootCA, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf" -pe -r -sky exchange -cy authority -sk PrivateKey -eku 1.3.6.1.5.5.7.3.2,1.3.6.1.5.5.7.3.1 -ss Root -sr LocalMachine -m 2 WcfBridge_RootCA.cer
+```
```
Comment (roncain) [inline src/System.Private.ServiceModel/tools/test/SelfHostWcfService/Certificates/README.md]: Nit: could you add a comment how the -eku values were obtained?

File: src/System.Private.ServiceModel/tools/test/SelfHostWcfService/Certificates/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+The certificate gets placed into the *Local Machine*'s *Root Certificate* store, as well as in the current directory's *WcfBridge_RootCA.cer*. 
+
+> *Optional:* If it's necessary to export the private key, you can do it using certmgr.msc:
+>
```
Comment (roncain) [inline src/System.Private.ServiceModel/tools/test/SelfHostWcfService/Certificates/README.md]: Nit: is it optional for us?  Ideally these steps are a script someone will follow to update the certificates, and they will want to know what steps are mandatory and which are not.

**Comment by roncain** [comment]
LGTM.  I pulled the PR, buddy-built it, and all InnerLoop and OuterLoop tests passed.
Thanks for solving this @iamjasonp.

File: src/System.Private.ServiceModel/tools/test/SelfHostWcfService/Certificates/README.md (hunk 2 above)
Comment (iamjasonp) [inline src/System.Private.ServiceModel/tools/test/SelfHostWcfService/Certificates/README.md]: Ok, I'll get rid of the "optional" steps listed.

File: src/System.Private.ServiceModel/tools/test/SelfHostWcfService/Certificates/README.md (hunk 1 above)
Comment (iamjasonp) [inline src/System.Private.ServiceModel/tools/test/SelfHostWcfService/Certificates/README.md]: Done - how's this?

**Comment by iamjasonp** [comment]
There's still a problem with the use of the online CRL; I'm trying to see if `makecert -crl` can solve our problems there by specifying a CRL to place into the root store. 

With this cert, if we can disable the CRL online revocation check, the TCP security tests pass.

**Pull request merged**